src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/MessageService.java

```diff
@@ -387,4 +391,69 @@ private MessageNonOCDS getMessageNonOCDS(
             receiver -> new OrganizationReference1().id(receiver.getId()).name(receiver.getName()))
             .collect(Collectors.toList()));
   }
+
+    public List<Message> getMessageListByEeventId(MessageRequestInfo messageRequestInfo, String messageId,String page , String pageSize) {
+      var jaggaerUserId = userProfileService
+              .resolveBuyerUserProfile(messageRequestInfo.getPrincipal())
+              .orElseThrow(() -> new AuthorisationFailureException(JAGGAER_USER_NOT_FOUND)).getUserId();
+      var event = validationService.validateProjectAndEventIds(messageRequestInfo.getProcId(),
+              messageRequestInfo.getEventId());
+      var messagesResponse =
+              jaggaerService.getMessages(event.getExternalEventId(), Integer.valueOf(pageSize));
+
+      var allJaggerMessages = messagesResponse.getMessageList().getMessage().stream().filter(message -> message.getMessageId()
+                                              == Integer.valueOf(messageId)).collect(Collectors.toList());
+      var evntNameSplit = messageRequestInfo.getEventId().split("-");
+      var tenderEventId = evntNameSplit[evntNameSplit.length -1];
+      var allTenderDbMessageAsyncStream = retryableTendersDBDelegate.getMessagesByEventId(Integer.valueOf(tenderEventId)).stream().filter(message -> message.getMessageId() == Integer.valueOf(messageId) && message.getStatus() == MessageTaskStatus.INPROGRESS);
```

Comment:
This message id is not the message id from Jagger. It's unique id for the message task. This condition will never be true.